packages/shared/client/ReplayClient.ts

```diff
@@ -395,12 +433,36 @@ export class ReplayClient implements ReplayClientInterface {
     return { events, requests };
   }
 
+  async findPaints(): Promise<TimeStampedPointWithPaintHash[]> {
+    const sessionId = await this.waitForSession();
+
+    const sortedPaints: TimeStampedPointWithPaintHash[] = [{ point: "0", time: 0, paintHash: "" }];
+
+    const onPaints = async ({ paints }: paintPoints) => {
+      paints.forEach(async ({ point, time, screenShots }) => {
+        const paint: TimeStampedPointWithPaintHash = {
+          paintHash: screenShots.find(({ mimeType }) => mimeType == "image/jpeg")?.hash ?? "",
+          point,
+          time,
+        };
+
+        insert(sortedPaints, paint, (a, b) => a.time - b.time);
```

Comment:
The main argument for sorting while in progress is that if we ever wanted to restore the streaming behavior, it would already work without requiring additional changes. Otherwise I think this is mostly just a matter of preference, and should not have a big performance impact either way.